frontend/package.json

```diff
@@ -46,6 +46,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.12.3",
```

Comment:
can you just remove this dependency real quick I don't think it's needed

Reply:
Removed